be/src/exec/pipeline/pipeline_driver_executor.cpp

```diff
@@ -346,7 +346,13 @@ void GlobalDriverExecutor::report_exec_state(QueryContext* query_ctx, FragmentCo
     // if it is done exec state report, We need to ensure that this report is executed with priority
     // and is retried as much as possible to ensure success.
     // Otherwise, it may result in the query or ingestion status getting stuck.
-    this->_exec_state_reporter->submit(std::move(report_task), done);
```

Comment:
What if this retry still failed?
This code looks strange. better to add retreat strategy inside the lambda